drivers/usb/device/usb_dc_stm32.c

```diff
@@ -36,15 +36,26 @@ LOG_MODULE_REGISTER(usb_dc_stm32);
 #error "Only one interface should be enabled at a time, OTG FS or OTG HS"
 #endif
 
+/*
+ * Vbus sensing is determined based on the presence of the designated pin(s) in the device tree.
```

Comment:
It should be stated more clearly that we're checking if the pin is actually used.
This is not obvious. This only works thanks to the `/omit-if-no-ref/` specifier which makes `usb_otg_hs_vbus_pxx` nodes present only if they are used.

Reply:
Do you find the description any clearer with the last change?